webextensions/api/identity.json

```diff
@@ -45,6 +51,10 @@
                   "version_added": "53",
                   "version_removed": "63",
                   "notes": "The <code>redirect_uri</code> URL parameter is required."
+                },
+                {
+                  "version_added": "75",
+                  "notes": "The <code>redirect_uri</code> URL parameter must be set to the URL returned by <code>identity.getRedirectURL()</code>."
```

Comment:
With the lot of comments about `redirect_url`, how about adding a new row? `redirect_url`.

Note that Chrome, Opera and Edge does not support the property; their documentation is at https://developer.chrome.com/docs/extensions/reference/identity/#type-WebAuthFlowDetails